.github/workflows/npm-publish.yml

```diff
@@ -33,12 +33,14 @@ jobs:
         run: pnpm i
       - name: Run postinstall script for dependencies
         run: pnpm rb
+      - name: Install npm
+        # TODO: OIDC requires npm >=11.5.1.
+        # Until Node.js v24 is LTS (with npm 11 as the default), we need to bump.
```

Comment:
https://github.com/DIYgod/RSSHub/actions/runs/19705948126/job/56453457811#step:4:20

24 is already LTS.